_posts/2019-09-05-cover-song-similarity.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Cover song similarity algorithms in Essentia
+category: news
+---
+
+
+`Cover Song Identification` or `Music Version Identification` is a task of identifying when two musical recordings are derived from the same music composition. A cover version of a song can be drastically different from the original recording. It may have variations in key, tempo, structure, melody, harmony, timbre, language and lyrics compared to its original version, which makes it a challenging task [1]. It is typically set as a query-retrieval task where the system retrieves a ranked list of possible covers of a given query song from a reference database. We recommend you reading [1] for more comprehensive details of this task. 
+
+Most of the state-of-the-art cover identification systems use pairwise comparison of two musical recordings to compute a similarity distance by using the following workflow.
+
+
+- Tonal feature extraction using chroma features such as [`HPCP`](https://essentia.upf.edu/reference/std_HPCP.html).
+- Post-processing of the tonal features to achieve various invariance to musical facets (eg. Pitch invariance [5]).
+- Computing cross-similarity between a pair of query and reference song tonal features [2, 4].
+- Local sub-sequence alignment to compute the pairwise cover song similarity distance [2, 3, 6].
```

Comment:
what does the "pairwise" mean in this context and what is the goal of using this word?

Reply:
Without knowing the algorithm in advance, the may be unclear for a new reader.

Reply:
May be we want to have an introductory phrase saying that the algorithm extracts a sequence of tonal features, and then compute the similarity between those sequences.

Reply:
The thing is that the algorithm we provide doesn't extract features. It just computes the similarity between a given pair of input features. Anyway, I improved some parts of the introduction to give a bit more clear idea about the process.